core/src/main/java/org/keycloak/jose/jwk/ECPublicJWK.java

```diff
@@ -17,6 +17,7 @@
 
 package org.keycloak.jose.jwk;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.keycloak.common.util.PemUtils;
```

Comment:
L22 and L24's imported class are no longer used.

Reply:
Updated, Thanks for the suggestion